### PR TITLE
Update expired Discord link + typo

### PR DESCRIPTION
**Pull request by omahs**
Update expired Discord link
+ correcting 1 typo

### CODE REVIEW

File: docs/overview/what-is-cyberconnect.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ description: The purpose of the CyberConnect Protocol is to empower users to tak
 
 ## Introduction
 
-CyberConnect is a decentralized [social graph](/concepts/social-graph/) protocol that helps dApps bootstrap network effects and build personalized social experiences. The decentralized protocol allows users to own their social connections while providing a composable social data layer for developers.
+CyberConnect is a decentralized [social graph](/concepts/social-graph/) protocol that helps dApps bootstrap network effects and builds personalized social experiences. The decentralized protocol allows users to own their social connections while providing a composable social data layer for developers.
```
Comment (HappySean2845) [inline docs/overview/what-is-cyberconnect.md:12]: `build` seems right? 
`dApps build`

File: docs/overview/build-with-cyberconnect.mdx [hunk 2]
```diff
@@ -19,7 +19,7 @@ import UsecaseCards from "@site/src/components/UsecaseCards";
 <UsecaseCards />
 
 ## Idea Pool
-We are integrating 70+ projects into the CyberConnect ecosystem. Here, we have curated a list of them as a source of inspiration for you. If you're ready to build with us or you'd like to be featured in the project showcase, feel free to reach us on [Discord](https://discord.gg/c7YPrT9P).
+We are integrating 70+ projects into the CyberConnect ecosystem. Here, we have curated a list of them as a source of inspiration for you. If you're ready to build with us or you'd like to be featured in the project showcase, feel free to reach us on [Discord](https://cyberconnect.fyi/discord).
```
Comment (HappySean2845) [inline docs/overview/build-with-cyberconnect.mdx:22]: Should be `https://discord.gg/cyberconnect`

File: i18n/zh/docusaurus-plugin-content-docs/current/overview/build-with-cyberconnect.mdx [hunk 3]
```diff
@@ -24,7 +24,7 @@ import UsecaseCards from "@site/src/components/UsecaseCards";
 
 ## 创意池
 
-我们正在将 70 多个项目整合到 CyberConnect 生态系统中。我们策划了一份清单，作为您的灵感来源。如果已准备好与我们一起构建，或者希望在这里展示您的项目，请随时通过 [Discord](https://discord.gg/c7YPrT9P) 与我们联系。
+我们正在将 70 多个项目整合到 CyberConnect 生态系统中。我们策划了一份清单，作为您的灵感来源。如果已准备好与我们一起构建，或者希望在这里展示您的项目，请随时通过 [Discord](https://cyberconnect.fyi/discord) 与我们联系。
```
Comment (HappySean2845) [inline i18n/zh/docusaurus-plugin-content-docs/current/overview/build-with-cyberconnect.mdx:27]: Should be `https://discord.gg/cyberconnect`

File: docs/overview/what-is-cyberconnect.md (hunk 1 above)
Comment (omahs) [reply docs/overview/what-is-cyberconnect.md:12]: @HappySean2845 I'll update the discord link and remove this 👍

**Comment by omahs** [comment]
I'll close this since the docs have changed

**Pull request closed without merging**